frontend/src/app/main/ui/workspace/colorpicker.cljs

```diff
@@ -346,7 +346,7 @@
                                       [:div {:class (stl/css :picker-detail-wrapper)}
                                        [:div {:class (stl/css :center-circle)}]
                                        [:canvas#picker-detail {:class (stl/css :picker-detail) :width 256 :height 140}]]
-                                      [:& ramp-selector
+                                      [:> ramp-selector*
```

Comment:
All `defc` components should not have a `*` suffix, and be called with `:>`. The `:&` syntax is deprecated. This way we avoid an automatic conversion between clojure and js formats that had an important performance cost.